src/DXRMapFixups/DeusEx/Classes/DXRFixupM03.uc

```diff
@@ -619,6 +627,20 @@ function AnyEntryMapFixes()
     switch(dxr.localURL) {
     case "03_NYC_747":
         SetTimer(1, true);
+        FixConversationFlagJump(GetConversation('AnnaEntrance'), 'AnnaThanks_Played', false, 'AnnaThanksChatDone', false);
+
+        // change the expiration of 'MeetLebedev2_Played'
+        c = GetConversation('MeetLebedev2');
+        cesf = new(c) class'ConEventSetFlag';
+        cesf.eventType = ET_SetFlag;
+        cesf.flagRef = new(c) class'ConFlagRef';
+        cesf.flagRef.flagName = 'MeetLebedev2_Played';
+        cesf.flagRef.value = true;
+        cesf.flagRef.expiration = 5;
```

Comment:
Are you certain this actually works?  Wouldn't the expiration be overwritten by the normal flag setting that happens at the end of the conversation?

I think it's still better if you use a new, unique flag.